### PR TITLE
Added Project Type new doctype to be added as link field for Project Type field in project instead of select field

### PR DESCRIPTION
**Pull request by KanchanChauhan**
- patch to add select items in project_type field as "Project Type"

screen shot 2017-07-18 at 5 41 54 pm

Fixed https://github.com/frappe/frappe/issues/3691

### CODE REVIEW

**Comment by mbauskar** [comment]
@KanchanChauhan,

Thanks for the PR,

Can you add the fixtures for project type for the new site the "External", "Internal", "Other" should be already created automatically (please check the install_fixture.py)

Also, project type **External** is used in Sales Order and email digest and user should not be able to delete **External** project type.

Closing the PR for now, You can reopen the PR once the changes are completed

**Comment by KanchanChauhan** [comment]
@mbauskar Done. Can you check now please.

File: erpnext/patches/v8_5/update_existing_data_in_project_type.py [hunk 1]
```diff
@@ -0,0 +1,19 @@
+# Copyright (c) 2017, Frappe and Contributors
+# License: GNU General Public License v3. See license.txt
+
+from __future__ import unicode_literals
+import frappe
+
+def execute():
+	frappe.reload_doctype("Project Type")
```
Comment (mbauskar) [inline erpnext/patches/v8_5/update_existing_data_in_project_type.py]: use `frappe.reload_doc("projects", "doctype", "project_type")` instead

File: erpnext/projects/doctype/project/project.json [hunk 2]
```diff
@@ -97,7 +97,7 @@
    "no_copy": 0, 
    "oldfieldname": "project_type", 
    "oldfieldtype": "Data", 
-   "options": "Internal\nExternal\nOther", 
+   "options": "Project Type",
```
Comment (mbauskar) [inline erpnext/projects/doctype/project/project.json]: Please also update the modified date for project doctype

**Review by mbauskar — CHANGES_REQUESTED** [review]
Please see my comments,

Needs rebase !!

**Review by mbauskar — APPROVED** [review]

**Pull request merged**